lib/plugin.js

```diff
@@ -159,6 +159,12 @@ class ReCaptcha {
 }
 
 export default function (_, inject) {
+  const { recaptcha = {} } = _.$config
```

Comment:
```suggestion
  const { recaptcha = {} } = _.$config || {}
```

For retro-compatibility